themes/bootstrap3/templates/RecordTab/holdingsils/standard.phtml

```diff
@@ -17,12 +17,21 @@
         <?=$this->transEsc('On Reserve - Ask at Circulation Desk')?><br>
       <?php endif; ?>
       <?php
-        $statusClass = $this->availabilityStatus()->getClass($availabilityStatus);
-        $statusDescription = $availabilityStatus->getStatusDescription();
-        $schemaAvailabilityUri = $availabilityStatus->getSchemaAvailabilityUri();
+        if ($holding['in_transit'] ?? null) {
+          $statusClass = 'text-danger';
+          $statusDescription = empty($holding['in_transit_date']) ? 'transit_to' : 'transit_to_date';
+          $statusDescriptionTokens = ['%%location%%' => $holding['in_transit'], '%%date%%' => $holding['in_transit_date'] ?? ''];
+          $schemaAvailabilityUri = $availabilityStatus->getSchemaAvailabilityUri();
```

Comment:
This line should be in the else clause, and the URI from the else clause should be up here. When I was putting together my example, I must have pasted into the wrong clause -- sorry about that!